regex/default.json5

```diff
@@ -1,4 +1,7 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
-  extends: ["github>strg-at/renovate-config//regex/opensearch.json5"],
+  extends: [
+    "github>strg-at/renovate-config//regex/opensearch.json5",
+    "github>strg-at/renovate-config//regex/debian-packages.json5",
```

Comment:
either you extend it here or you add it to the repo. if you add it here, it will be loaded by default in all repos as its part of base.json5. if you extend it in the repo it will just run there.